pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go

```diff
@@ -35,7 +35,7 @@ type responses struct {
 	branches     []getBranchDataRepositoryRefsRefConnection
 	prs          []getPullRequestDataRepositoryPullRequestsPullRequestConnection
 	page         int
-	contribs     []*github.Contributor
+	contribs     [][]*github.Contributor
```

Comment:
slice of slice?

Reply:
I updated the test to use this version instead, it still returns one slice at a time since we use the `page` variable to index it and get the response per that page

Reply:
yea i was referring to this: https://github.com/liatrio/liatrio-otel-collector/blob/main/pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers.go#L209 the code just appends

Reply:
That still makes sense, each response will contain a list of contributors that will be appended and after all the pages are appended we'll be left with all the contributors